app/assets/javascripts/search.coffee

```diff
@@ -21,47 +21,56 @@ $(document).ready ->
     else
       $('#lightbox').remove()
 
-# Handle presence of Javascript by turning off or on visibility of JS sensitive objects
-$('.js-disabled-object').css 'visibility': 'hidden'
-$('.js-enabled-object').css 'visibility': 'visible'
+  # Handle presence of Javascript by turning off or on visibility of JS sensitive objects
+  $('.js-disabled-object').css 'visibility': 'hidden'
+  $('.js-enabled-object').css 'visibility': 'visible'
 
-# On search caret click, show search dropdown
-$ ->
+  # On search caret click, show search dropdown
   $('#search-drop').on 'click', ->
     if $('.search-dropdown').css('display') == 'none'
       $('.search-dropdown').css 'display': 'block'
       $('#search-drop-caret').css 'transform': 'rotate(180deg)'
       $('#search-submit-button').css 'border-bottom': '1px solid #95a5a6'
-
     else
       $('.search-dropdown').css 'display': 'none'
       $('#search-drop-caret').css 'transform': 'rotate(0deg)'
       $('#search-submit-button').css 'border-bottom': '1px solid transparent'
-$ ->
-  $('#record_state_published_records').on 'click', ->
-    $("#search-text").text('Search Collections')
 
-  $('#record_state_variables').on 'click', ->
-    $("#search-text").html('<span style="padding-right:11px;">Search Variables</span>')
+  if $('#search-submit-button').length > 0
+    recordType = $('input[name="record_type"]:checked').val()
+    $searchButtonSpan = $('#search-submit-button-text')
+
+    switch recordType
```

Comment:
My thinking was that since we will likely need to add a `services` option, that using `case/switch` might be slightly preferable. Would you rather I changed it to what you suggested?